src/moz_build_info.js

```diff
@@ -20,7 +24,12 @@ export function archiveFiletype(os) {
  */
 export function archiveFileSuffix(product, os) {
   let ospart = (product === 'firefox' && os === 'mac64') ? 'mac' : os;
-  return `${ospart}.${archiveFiletype(os)}`;
+  if (product.indexOf('emulator') != -1) {
```

Comment:
nit: Prefer `!==` to `!=`. This is a good resource http://bonsaiden.github.io/JavaScript-Garden/#types.equality